pentaho-js/src/main/javascript/cdf/components/SchedulePrptComponent.js

```diff
@@ -82,10 +84,17 @@ define([
       this.scheduleParameters = this.scheduleParameters || {};
       this.scheduleParameters["jobParameters"] = jobParameters;
       var success = false;
+      var protectedUrl = SchedulePrptComponentExt.getScheduledJob();
+      var csrfToken = csrfClient.getToken(protectedUrl);
+      var headers = {};
+      if(csrfToken !== null) {
+        headers[csrfToken.header] = csrfToken.token;
+      }
       $.ajax({
         url: SchedulePrptComponentExt.getScheduledJob(),
         async: false,
         type: "POST",
+        headers: headers,
```

Comment:
Like commented on the legacy case, `url` should receive the value of `protectedUrl`.

Reply:
addressed review comments